index.html

```diff
@@ -179,17 +179,9 @@ <h2 id="subscribe">Developer digest</h2>
                         </div>
                     </form>
                 </div>
-                <div class="archive">
-                    <ul>
-                        <li>Archives:</li>
-                        <li><a href="https://stripemail.createsend.com/campaigns/reports/viewCampaign.aspx?d=d&c=7CA158CD0A57402E&ID=94EC6240E9E286EC2540EF23F30FEDED&temp=False&tx=0" target="_blank">May 2019</a></li>
-                        <li><a href="https://stripemail.createsend.com/campaigns/reports/viewCampaign.aspx?d=d&c=7CA158CD0A57402E&ID=E5495BB01A8E19B72540EF23F30FEDED&temp=False&tx=0" target="_blank">March 2019</a></li>
-                        <li><a href="https://stripemail.createsend.com/campaigns/reports/viewCampaign.aspx?d=d&c=7CA158CD0A57402E&ID=17A9814FC873EAAC2540EF23F30FEDED&temp=False&tx=0" target="_blank">February 2019</a></li>
-                        <li><a href="https://stripemail.createsend.com/campaigns/reports/viewCampaign.aspx?d=d&c=7CA158CD0A57402E&ID=A8E0AF2FC66664BB2540EF23F30FEDED&temp=False&tx=0" target="_blank">November 2018</a></li>
-                        <li><a href="https://stripemail.createsend.com/campaigns/reports/viewCampaign.aspx?d=d&c=7CA158CD0A57402E&ID=A3C45E8E583601BC2540EF23F30FEDED&temp=False&tx=0" target="_blank">August 2018</a></li>
-                        <li><a href="https://stripemail.createsend.com/campaigns/reports/viewCampaign.aspx?d=d&c=7CA158CD0A57402E&ID=ADA3E1BAB85DEDCA2540EF23F30FEDED&temp=False&tx=0" target="_blank">June 2018</a></li>
-                    </ul>
-                </div>
+                <p>
+                    Read previous issues on <a href="https://dev.to/t/stripedevdigest" target="_blank">DEV.to</a>
```

Comment:
Could you swap "Read previous issues on" with "Read recent digests on" ? 
_(since issues is so closely tied to GH repos)_

Reply:
![image](https://user-images.githubusercontent.com/23213994/85193519-dcd98600-b2fb-11ea-84aa-84b2c723747a.png)